lib/grape/validations/types/primitive_coercer.rb

```diff
@@ -12,25 +12,29 @@ class PrimitiveCoercer < DryTypeCoercer
         MAPPING = {
           Grape::API::Boolean => DryTypes::Params::Bool,
           BigDecimal => DryTypes::Params::Decimal,
+          Numeric => DryTypes::Params::Integer | DryTypes::Params::Float | DryTypes::Params::Decimal,
 
           # unfortunately, a +Params+ scope doesn't contain String
           String => DryTypes::Coercible::String
         }.freeze
 
         STRICT_MAPPING = {
           Grape::API::Boolean => DryTypes::Strict::Bool,
-          BigDecimal => DryTypes::Strict::Decimal
+          BigDecimal => DryTypes::Strict::Decimal,
+          Numeric => DryTypes::Strict::Integer | DryTypes::Strict::Float | DryTypes::Strict::Decimal
         }.freeze
 
         def initialize(type, strict = false)
           super
 
           @type = type
 
+          # TODO: use const_get(.., false) to prevent vague errors on unknown types, but it breaks a lot of other stuff
```

Comment:
Can we just revert this part of the change and open a new issue on GH please?

Reply:
Done